lua/starfall/libs_sv/wire.lua

```diff
@@ -308,6 +308,43 @@ function wire_library.adjustOutputs(names, types)
 	WireLib.AdjustSpecialOutputs(ent, names, types)
 end
 
+--- Creates/Modifies wire inputs/outputs. All wire ports must begin with an uppercase
+-- letter and contain only alphabetical characters.
```

Comment:
They can end with a number as well

Reply:
I just copied and modified that from adjustInputs and adjustOutputs functions, so I guess that should be a different commit

Reply:
I don't mind if you correct both